.gitmodules

```
@@ -1,7 +1,7 @@
 [submodule "spack"]
   path = spack
   url = https://github.com/jcsda/spack
-  branch = release/1.9.0
+  branch = spack-stack-1.9.2rc2
```

```suggestion
  branch = release/1.9.0
```
Let's skip this for simplicity. We only really need to tag the releases for the submodules